addon/data/shoot-panel.js

```diff
@@ -89,6 +89,12 @@ class ShareButtons extends React.Component {
       <a href="#" onClick={ this.props.onCopyClick }>
         <img src={ `icons/link-${size}.png` } />
       </a>
+      <a href="#" onClick={ this.props.onCopyImageClick }>
```

Comment:
This will probably update location.hash to be `#`, which probably isn't meaningful in this case, but generally I like to avoid that.  Fixing it either requires a change to `onCopyClick` or turning this into something other than an anchor (plus CSS)